CONTRIBUTING.MD

```diff
@@ -0,0 +1,35 @@
+## Contributing
+
+You are more than welcome to make any contributions.
+Please create Pull Request for any changes.
+
+### Fork the Repository
```

Comment:
This should be `setting up development environment` instead. You should also mention third party brokers/backends that need to be installed here.